examples/CivisML_parallel_training.ipynb

```diff
@@ -0,0 +1,682 @@
+    "\n",
+    "- There are several pre-defined model types you can use to create a pipeline (check the docs above for a full list).\n",
+    "- sparse_logistic will create an L1 penalized logistic regression for variable selection, then build an unpenalized logistic regression from the selected variables\n",
+    "- I need to explicitly pass which column is the dependent variable\n",
+    "\n",
+    "I'll initialize the pipelines below"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "workflows = ['sparse_logistic',\n",
+    "             'random_forest_classifier',\n",
+    "             'extra_trees_classifier']\n",
+    "\n",
+    "models = []\n",
+    "for wf in workflows:\n",
+    "    model = ModelPipeline(model=wf,\n",
+    "                          dependent_variable=\"is_cancer\",\n",
+    "                          model_name=wf)\n",
+    "    models.append(model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Trying different hyperparameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another thing I'm often doing is building lots of models with tweaked hyperparameters, to see what fits best.\n",
+    "\n",
+    "Let's add a bunch of gradient_boosted_classifiers with different params from the defaults.\n",
+    "\n",
+    "The code below passes a dict of hyperparameter choices to the ModelPipeline. **CivisML will automatically try all combinations of hyperparameter choices, and report back results from the best one** on the cross-validated data set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "cv_params = {'learning_rate': [0.01, 0.2],\n",
+    "            'n_estimators': [50, 200],\n",
+    "            'max_depth': [1, 2]}\n",
+    "model = ModelPipeline(model='gradient_boosting_classifier',\n",
+    "                      dependent_variable=\"is_cancer\",\n",
+    "                      cross_validation_parameters=cv_params,\n",
+    "                      model_name='best_gbc_from_hyperparam_search')\n",
+    "models.append(model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Creating your own estimator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Last but not least, I might want to roll my own classifier, apart from CivisML builtins. The CivisML docker imager allows me to insert my own classifiers from scikit-learn and other popular open source packages. You can see a list of them [here](https://civis-python.readthedocs.io/en/v1.5.2/ml.html#custom-models). It's easy to take your existing code and still get the benefits of CivisML parallelizability and metrics reporting.\n",
```

Comment:
I guess a better question is why do we need to mention the CivisML Docker image?  Maybe better just to point to the link and say that CivisML works with any of the models listed there.